config/replacements.php

```diff
@@ -151,112 +151,138 @@
-    'r\\ZendXml\\' => 'r\\ZendXml\\',
-    's\\ZendXml\\' => 's\\ZendXml\\',
-    't\\ZendXml\\' => 't\\ZendXml\\',
-    'u\\ZendXml\\' => 'u\\ZendXml\\',
-    'v\\ZendXml\\' => 'v\\ZendXml\\',
-    'w\\ZendXml\\' => 'w\\ZendXml\\',
-    'x\\ZendXml\\' => 'x\\ZendXml\\',
-    'y\\ZendXml\\' => 'y\\ZendXml\\',
-    'z\\ZendXml\\' => 'z\\ZendXml\\',
+    // class ZendAcl in zend-expressive-authorization-acl
+    'ZendAcl' => 'LaminasAcl',
+    'Acl\\ZendAcl' => 'Acl\\LaminasAcl',
+    'Acl\\\\ZendAcl' => 'Acl\\\\LaminasAcl',
+    'Zend\\Expressive\\Authorization\\Acl\\ZendAcl' => 'Mezzio\\Authorization\\Acl\\LaminasAcl',
+    'Zend\\\\Expressive\\\\Authorization\\\\Acl\\\\ZendAcl' => 'Mezzio\\\\Authorization\\\\Acl\\\\LaminasAcl',
+    // class ZendHttpClientDecorator in zend-feed
+    'ZendHttp' => 'LaminasHttp',
+    // class ZendModuleProvider in zend-config-aggregator-modulemanager
+    'ZendModule' => 'LaminasModule',
+    // class ZendRbac in zend-expressive-authorization-rbac
+    'ZendRbac' => 'LaminasRbac',
+    'Rbac\\ZendRbac' => 'Rbac\\LaminasRbac',
+    'Rbac\\\\ZendRbac' => 'Rbac\\\\LaminasRbac',
+    'Zend\\Expressive\\Authorization\\Rbac\\ZendRbac' => 'Mezzio\\Authorization\\Rbac\\LaminasRbac',
+    'Zend\\\\Expressive\\\\Authorization\\\\Rbac\\\\ZendRbac' => 'Mezzio\\\\Authorization\\\\Rbac\\\\LaminasRbac',
+    // class ZendRouter in zend-expressive-router-zendrouter
+    'ZendRouter' => 'LaminasRouter',
+    'Router\\ZendRouter' => 'Router\\LaminasRouter',
+    'Router\\\\ZendRouter' => 'Router\\\\LaminasRouter',
+    'Zend\\Expressive\\Router\\ZendRouter' => 'Mezzio\\Router\\LaminasRouter',
+    'Zend\\\\Expressive\\\\Router\\\\ZendRouter' => 'Mezzio\\Router\\\\LaminasRouter',
+    // class ZendViewRenderer in zend-expressive-zendviewrenderer
+    'ZendViewRenderer' => 'LaminasViewRenderer',
+    'ZendView\\ZendViewRenderer' => 'LaminasView\\LaminasViewRenderer',
+    'ZendView\\\\ZendViewRenderer' => 'LaminasView\\\\LaminasViewRenderer',
+    'Zend\\Expressive\\ZendView\\ZendViewRenderer' => 'Mezzio\\LaminasView\\LaminasViewRenderer',
+    'Zend\\\\Expressive\\\\ZendView\\\\ZendViewRenderer' => 'Mezzio\\LaminasView\\\\LaminasViewRenderer',
+    'a\\Zend' => 'a\\Zend',
+    'b\\Zend' => 'b\\Zend',
+    'c\\Zend' => 'c\\Zend',
+    'd\\Zend' => 'd\\Zend',
+    'e\\Zend' => 'e\\Zend',
+    'f\\Zend' => 'f\\Zend',
+    'g\\Zend' => 'g\\Zend',
+    'h\\Zend' => 'h\\Zend',
+    'i\\Zend' => 'i\\Zend',
+    'j\\Zend' => 'j\\Zend',
+    'k\\Zend' => 'k\\Zend',
+    'l\\Zend' => 'l\\Zend',
+    'm\\Zend' => 'm\\Zend',
+    'n\\Zend' => 'n\\Zend',
+    'o\\Zend' => 'o\\Zend',
+    'p\\Zend' => 'p\\Zend',
+    'q\\Zend' => 'q\\Zend',
+    'r\\Zend' => 'r\\Zend',
+    's\\Zend' => 's\\Zend',
+    't\\Zend' => 't\\Zend',
+    'u\\Zend' => 'u\\Zend',
+    'v\\Zend' => 'v\\Zend',
+    'w\\Zend' => 'w\\Zend',
+    'x\\Zend' => 'x\\Zend',
+    'y\\Zend' => 'y\\Zend',
+    'z\\Zend' => 'z\\Zend',
+    'a\\ZendOAuth' => 'a\\ZendOAuth',
```

Comment:
All these from a\\ZendOAuth are to z\\ZendXml are redundant now, as we have just [a-z]\\Zend without trailing namespace separator

Reply:
Looks like length of key _does_ kick in here with regards to `strtr()` (based on 3v4l that @michalbundyra created): https://3v4l.org/dipMb

Reply:
Yes, but in that case it does not matter - as we do not want to have replaced these phrases, so still I believe we can remove all others and keep just `[a-z]\\Zend`